tsfresh/feature_extraction/data.py

```diff
@@ -291,6 +295,124 @@ def __len__(self):
         return sum(grouped_df.ngroups for grouped_df in self.grouped_dict.values())
 
 
+def window(seq, window_width):
```

Comment:
Either keep this "private" by prefixing it with "_" or we add it to the dataframe functions utils, if you think other people can also benefit from it.